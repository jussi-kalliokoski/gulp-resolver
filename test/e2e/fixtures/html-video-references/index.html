<!DOCTYPE html>
<html lang="en">
<head>
</head>
<body>
    <video src="video.mp4" poster="cat.jpg" />
</body>
</html>
