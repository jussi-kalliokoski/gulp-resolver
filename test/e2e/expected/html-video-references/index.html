<!DOCTYPE html>
<html lang="en">
<head>
</head>
<body>
    <video src="video-3ff6123f.mp4" poster="cat.4890123.jpg" />
</body>
</html>
